query-node/schema.graphql

```diff
@@ -12,32 +12,127 @@ type Block @entity {
   network: Network!
 }
 
+enum MembershipEntryMethod {
+  PAID
+  SCREENING
+  GENESIS
+}
+
 "Stored information about a registered user"
-type Member @entity {
+type Membership @entity {
   "MemberId: runtime identifier for a user"
   id: ID!
 
   "The unique handle chosen by member"
-  handle: String @unique @fulltext(query: "membersByHandle")
+  handle: String! @unique @fulltext(query: "membersByHandle")
 
   "A Url to member's Avatar image"
-  avatarUri: String
+  avatarUri: String!
 
   "Short text chosen by member to share information about themselves"
-  about: String
-
-  "Blocknumber when member was registered"
-  registeredAtBlock: Int!
+  about: String!
```

Comment:
The about field should also be nullable because the runtime allows it. Can we still have an empty string if we use `String!` ?

Reply:
Done. The exclamation mark means non-nullable - it doesn't prevent empty strings. I'll discuss it with Atlas team if it is desirable to ensure non-empty strings. If that's the case, we should probably be consistent about it everywhere a `String` is used.

Reply:
Confirmed from Atlas team that non-nullable string is the desired state and empty strings are ok there.